olive/evaluator/olive_evaluator.py

```diff
@@ -33,6 +34,8 @@
 
 logger = logging.getLogger(__name__)
 
+# TODO: remove
+
 
 class DummyDataloader(Dataset):
```

Comment:
>DummyDataloader [](http://example.com/codeflow?start=6&length=15)

do we still need it?

Reply:
Already removed since the latest commit. Please refresh it.